_guide/_pages/interviews/core-values.md

```diff
@@ -54,108 +54,61 @@ Begin by introducing yourself by saying this or something similar to it:
-    - If no: what challenges do you anticipate? What do you think you'll need to be successful?
-
-- [Have you ever faced any challenges with a coworker or manager?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.95oxlva5ufr7)
-    - If applicable: How did you resolve the matter?
+- [18F is a fully distributed, remote team. What steps would you take to ensure that you and your team are always in sync with each other?](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.kbq69akgkj00)
+    - (If the candidate only talks about communicating with other engineers) How do you know if you’re communicating well with designers, PMs, product owners, and other non-engineers on your team?
 
-- [What steps would you take to ensure that you and your team are always in sync with each other?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.95oxlva5ufr7)
-
-- [Can you describe a time or situation in which you saw or noted something that wasn’t working well and advocated for an improvement?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.95oxlva5ufr7)
+- [Can you describe a time or situation in which you saw or noted something that wasn’t working well and advocated for an improvement?](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.2jqlt72nbfcu)
     - If applicable: What challenges did you face in doing this?
     - If applicable: Did your efforts succeed or fail, and can you explain why?
     - If applicable: Did you solicit help from your colleagues and/or manager?
 
-- [What is your work ethic and how do you approach work?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.95oxlva5ufr7)
-
-- [What is your preferred work/management style?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.95oxlva5ufr7)
-
-- [When you learn something new, how do you share it with your team?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.95oxlva5ufr7)
-
-### Technology and open source
-
-Regardless of whether or not the candidate is applying for a software development position, 18F relies heavily on technology and open source to execute its mission. This section focuses on those skills and experience.
-
-- [Have you ever worked with open source before?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.3qcfg9pfhobk)
-    - If applicable: What were you involved with, and what was your role?
-    - If applicable: How did you contribute?
-
-- [Are you comfortable with sharing your work in a public fashion?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.3qcfg9pfhobk)
```

Comment:
this feels potentially useful for understanding how candidate's work in the open. Probably not useful for all candidates, but maybe good to keep in pocket.